doozer/doozerlib/backend/konflux_image_builder.py

```
@@ -651,6 +650,7 @@ async def _get_for_arch(arch):
 
         return all_package_nvrs, all_source_rpms
 
+    @retry(reraise=True, wait=wait_fixed(5), stop=stop_after_attempt(3))
     async def update_konflux_db(
```

Pylint (4.0.2)

[refactor] Too many arguments (7/5)

(R0913)